smart_alert.js

```diff
@@ -17,6 +17,14 @@
  * 3. btm-left 画面左下に表示します。
  * 4. btm-right 画面右下に表示します。
  */
+
```

Comment:
I would prefer to have this as a static property of the SmartAlert class, because I want it to be completed only within the scope of the class.

Reply:
Sorry, I didn't understand which part of the code you're referring to here. Could you clarify?

Reply:
Sorry, I made a mistake in commenting the wrong line.  I meant the variable 'alertsObj' on line 21.

Reply:
Thank you!
The source looks good to me, so I'll actually run it and merge it in if everything's okay!